src/languages/de.json

```diff
@@ -0,0 +1,101 @@
+{
+    "chat": {
+        "actions": {
+            "banUser": "Ban Benutzer",
```

Comment:
Well, I am not sure how to translate 'ban', I'd expect gamers to know what it means. 'Verbanne Benutzer' does not sound very beautiful, does it. 
As for the other ones, Admin and Moderator should be appropriate, the term 'Promote' should be an equivalent to 'make', in any other case I would need some help on finding the correct term.